### DISCUSSION TITLE
7th Community Meeting 27/10/2022

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#7314] [General]
Hello to all Arduino ESP32 Core Users 🚀 

**Meeting purpose,**
we appreciate all of your contribution and help and we would like to improve your User experience on this Project. 🤩

As a part of this improvement we would like to uncover our internal Development to you, have clear and transparent planning and also listen to your requests, ideas and valuable Feedback.

To fulfill this, we would like to held every Month Community Meeting where we can share main updates and also have an open discussion with you.

---

**💥Seventh Community Meeting will take place on Thursday 27/10/2022 at 4 pm GMT+2 time zone. (1 hour length) We will use Google Meet.**

---

We want to have some idea about how many of you would be interested in joining these Meetings so instead of sharing directly link to Online Meeting itself, it will work by invitation.

**If you want to be added to upcoming Meeting and receive invitation please send an email to [vojtech.bartoska@espressif.com](mailto:vojtech.bartoska@espressif.com).** 

If you already participated on any Community Meeting, you are automatically invited to upcoming Meeting and it's up to you if you join us again or not.

---

### Agenda will be specified below in the comments, feel free to add any topic or idea :) .

---

See you on the Meeting!

**Comment by mrengineer7777** [comment]
I unfortunately just missed the meeting yesterday, but I've very curious to hear what you guys are working on.

**Comment by VojtechBartoska** [comment]
Hello,

I'm sharing the slides from the Meeting. Should help with general overview @mrengineer7777.

[07_Seven_Community_meeting-Arduino-October_2022.pdf](https://github.com/espressif/arduino-esp32/files/9951583/07_Seven_Community_meeting-Arduino-October_2022.pdf)
